### PR TITLE
Theme2.1

### PR DESCRIPTION
**Pull request by dmartine**
Created a lab for adding a custom theme to WAb. Sorry for all the commits I was trying to format with each commits. Thanks to John G for showing me the mark up editor. Please squash this pull request to reduce the commits.

### CODE REVIEW

**Comment by jgravois** [comment]
we need to get rid of the `add_custom_theme/Corel Auto-Preserve/` folder

**Comment by dmartine** [comment]
Formatting is screwed up need to fix it the editor didn't catch it

File: develop/wab/add_custom_theme/lab.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+      ```CSS       
+        .jimu-title,
+        .jimu-subtitle {
+          color: #323232;
+        }
+      ```
+      * Remove the shadow and add a border.
+    
+    ```CSS
```
Comment (jgravois) [inline develop/wab/add_custom_theme/lab.md]: i'm sure you're sick of me by now, but this last .css code snippet doesn't have the same indentation as the others.

File: develop/wab/add_custom_theme/lab.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    
+    ```CSS                  
+      .jimu-widget-header-controller{
+        -moz-box-shadow: none;
+        -webkit-box-shadow: none;
+        box-shadow: none;
+        border-bottom: 2px solid #323232;
+      }
+    ```
+6. To see the changes you just implemented in the Web AppBuilder GUI select any theme other than Hackerlab Theme (e.g. Foldable theme) and click `Save`, then select the Hackerlab Theme. This refreshes the Custom Theme application by using the current theme selected.
```
Comment (jgravois) [inline develop/wab/add_custom_theme/lab.md]: for some reason, markdown switches from `6.` to `i.` here.

**Comment by jgravois** [comment]
67 commits later, LGTM.

**Pull request merged**